### DISCUSSION TITLE
Component red line in _app.tsx

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [vercel/next.js#44188] [Help] [answered]
https://github.com/vercel/next.js/blob/canary/examples/with-context-api/pages/_app.tsx

### Why it is happening ? How to solve it ? Do I need to solve it [😂](https://emojipedia.org/face-with-tears-of-joy/)

![Screenshot (119)](https://user-images.githubusercontent.com/71011043/208672818-bfd3a6c9-8a0e-4a92-ac47-ccefb1cf4aa6.png)

**Comment by icyJoseph** [comment, marked as answer]
What TypeScript version are you using? VSCode has its own TypeScript version, and you have to tell it to use the project's TypeScript version. https://stackoverflow.com/a/39676463 Read the last part of this StackOverflow answer.

That being said I did not see this error when starting the example. I'd recommend upgrading to "@types/react": "18.0.17" though, since the template is using already React 18. 

For what is worth this issue came about when upgrading types from 17 to 18.

**Comment by thanhtutzaw** [comment]
Thanks for the answer .
![Screenshot (120)](https://user-images.githubusercontent.com/71011043/208793573-68df369a-9ba8-4f96-b90f-36b2474e3694.png)
